bootstrap.go

```diff
@@ -22,10 +22,11 @@ import (
 	"runtime/debug"
 	"strings"
 
+	"github.com/juju/errors"
 	"github.com/ngaut/log"
 	"github.com/pingcap/tidb/mysql"
 	"github.com/pingcap/tidb/sessionctx/variable"
-	"github.com/pingcap/tidb/util/errors"
+	terrors "github.com/pingcap/tidb/util/errors"
```

Comment:
This will be removed after Zhou Sir's pr merged into master.
I have a TODO in this file.